lfucg-exactions/client/js/components/PaymentSummary.js

```diff
@@ -146,10 +145,10 @@ class PaymentSummary extends React.Component {
                                         </div>
                                         <div className="col-xs-12">
                                             <p className="col-xs-12">Lot Address: {payments.lot_id.address_full}</p>
-                                            <p className="col-md-4 col-xs-6">Current Exactions: {payments.lot_id && payments.lot_id.lot_exactions && payments.lot_id.lot_exactions.current_exactions}</p>
-                                            <p className="col-md-4 col-xs-6">Plat: {payments.lot_id.plat.name}</p>
-                                            <p className="col-md-4 col-xs-6 ">Lot Number: {payments.lot_id.lot_number}</p>
-                                            <p className="col-md-4 col-xs-6">Permit ID: {payments.lot_id.permit_id}</p>
+                                            <p className="col-xs-6">Current Exactions: {payments.lot_id && payments.lot_id.lot_exactions && payments.lot_id.lot_exactions.current_exactions}</p>
+                                            <p className="col-xs-6">Plat: {payments.lot_id.plat.cabinet}-{payments.lot_id.plat.slide}</p>
```

Comment:
Check for plat before calling cabinet, as cabinet could be blank.

Reply:
Cabinet should be there regardless on the state tree... yes? It come in as blank in other instances where there is no value to display. IE. plat.block or something of the sort.